theme_paptic/views/customizations.xml

```diff
@@ -206,7 +206,7 @@
 <!-- ==== Three Columns ===== -->
 <template id="s_three_columns" inherit_id="website.s_three_columns" name="Paptic s_three_columns">
     <xpath expr="//section" position="attributes">
-        <attribute name="class" remove="bg-200 pt32 pb32 o_cc o_cc3" add="pt120 pb120" separator=" "/>
+        <attribute name="class" remove="pt32 pb32 o_cc o_cc3 o_cc2" add="pt120 pb120" separator=" "/>
```

Comment:
`Remove o_cc3 o_cc2` ? Seems like a mistake that was done before, should be fixed here :)